scm/github.go

```diff
@@ -595,7 +607,7 @@ func (s *GithubSCM) CreateGroup(ctx context.Context, opt *TeamOptions) (*Reposit
 			Message: fmt.Sprintf("%+v", opt),
```

Comment:
Does this method need to return both a `Repository` and `Team`? Could we get away with returning only one of them?

Reply:
We need to save GitHub repository ID and GitHub team ID into the database, so I'm not sure if we can.